AUTHORS

```diff
@@ -3432,5 +3432,8 @@ Alexandru Comanescu <comanescu.robert93@gmail.com>
 madflow <madflow@web.de>
 Austin Kelleher <austin.kell47@gmail.com>
 apeltop <psycho3081@gmail.com>
+Livia Medeiros <74449973+LiviaMedeiros@users.noreply.github.com>
+Nikolaos Papaspyrou <nikolaos@chromium.org>
+apeltop <sunshine@ptokos.com>
```

Comment:
Likely duplicate of `apeltop <psycho3081@gmail.com>`. If so, needs a `.mailmap` entry.